Makefile

```diff
@@ -0,0 +1,48 @@
+include .env
+
+default: up
+
+.PHONY: help
+ifneq (,$(wildcard docker.mk))
+help : docker.mk
+	@sed -n 's/^##//p' $<
+else
+help : Makefile
+	@sed -n 's/^##//p' $<
+endif
+
+## up : Start up containers, including the project_name defined at .env file.
+.PHONY: up
+up:
+	@echo "Starting up containers for $(PROJECT_NAME)..."
+	@docker compose -f docker-compose.nginx.yml -p nginx up -d
+	@docker compose up -d --remove-orphans
+	@echo "http://$(PROJECT_BASE_URL)"
+
+## install : Install default Open Social profile.
+.PHONY: install
+install:
+	@echo "Install Open Social profile on $(PROJECT_NAME)..."
+	@docker exec -i $(PROJECT_NAME)_web bash /var/www/scripts/social/install/install_script.sh
```

Comment:
Maybe it would be handy to pass extra params to the install script. So we could use `make install --with-optional` for example.